src/model.rs

```diff
@@ -27,10 +33,14 @@ pub struct Pod {
     /// Unique id based on reproducibility.
     #[serde(skip)]
     pub hash: String,
-    image: String,
-    command: String,
-    input_stream_map: BTreeMap<String, StreamInfo>,
-    output_dir: PathBuf,
+    /// Reproducible environment for compute.
+    pub image: String,
+    /// Space-delimited shell command to begin computation.
+    pub command: String,
+    /// A dictionary of named input streams.
+    pub input_stream_map: BTreeMap<String, StreamInfo>,
+    /// Absolute output directory within the environment.
+    pub output_dir: PathBuf,
     output_stream_map: BTreeMap<String, StreamInfo>,
     source_commit_url: String,
```

Comment:
We should put a comment on what this is and also discussed about it since it doesn't really affect repoduceabilty at all

Reply:
We decided on perhaps moving it to a specific PodAnnotation that is seperate from Annotation. Should make it optional